tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compileOnSave": true,
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es2017",
+    "lib": ["es2017"],
+    "typeRoots": ["node_modules/@types"],
+    "rootDir": "./lib",
+    "outDir": "./build",
+    "moduleResolution": "node",
+    "esModuleInterop": true,
+    "allowJs": true,
```

Comment:
Configuring JS compatibility mode for now, as there is no intention to rewrite the entire Dredd to TypeScript in a single pull request.